### PR TITLE
issue-14-docs-changes

### PR DESCRIPTION
**Pull request by CarlaLlama**

### CODE REVIEW

**Comment by CarlaLlama** [comment]
@hodgestar I can't think of anything else to add to readthedocs other than the code comments (which I still need to make for the most part!). Is there some sort of autodoc/autopy command to auto generate them like:
`.. autojs:: ../lib/interaction_machine.js` [from vumi sandbox]

**Comment by hodgestar** [comment]
There is! And it is built in to Sphinx -- http://www.sphinx-doc.org/en/stable/ext/autodoc.html :)

File: docs/usage.rst [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+To see list of all twistd plugins::
+
+	(ve)$ twistd -d
+
+In order to configure the proxy using default settings, a blacklist must be specified as follows::
+
+	(ve)$ twistd -n vumi_http_proxy --blacklist=BLACKLISTFILENAME
+
+This blacklist must be a PyYAML configuration file - an example of which can 
+be found in ./docs/proxy_blacklist.yml
```
Comment (hodgestar) [inline docs/usage.rst:21]: If you like you can include the example blacklist using a literalinclude:
http://www.sphinx-doc.org/en/stable/markup/code.html#directive-literalinclude

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**